src/codegen/fast_math.ispc

```diff
@@ -78,13 +79,9 @@ static const uniform float PX4expf = 4.1665795894E-2f;
 static const uniform float PX5expf = 1.6666665459E-1f;
 static const uniform float PX6expf = 5.0000001201E-1f;
 
-static const uniform float LOG2EF = 1.44269504088896341f;
+static const uniform float LOG2EF = 1.44269504088896341f; // 1/ln(2)
 
-/// double precision exp function
-static inline double vexp(double initial_x) {
-    double x = initial_x;
-    double px = dpfloor(LOG2E * x + 0.5d);
-    const int32 n = px;
+static inline double egm1(double x, double px) {
 
     x -= px * 6.93145751953125d-1;
     x -= px * 1.42860682030941723212d-6;
```

Comment:
maybe these two constants should be also named, liek they are for the single prec version
and the two lines should be taken out of this function because they are not part of the `$e^g -1$` expression. It makes more sense the way it's onde for single prec.